frontend_build/src/read_bundle_plugins.js

```diff
@@ -24,8 +24,6 @@ var libs = function(kolibri_name) {
     'logging': kolibri_name + '.lib.logging',
     'vue': kolibri_name + '.lib.vue',
     'kolibri': kolibri_name,
-    'core-base': kolibri_name + '.lib.coreBase',
```

Comment:
ok - I'll re-instate them.

I still don't fully understand how these functions and was just trying to clean up. It's not clear to me how these interact with webpack.config.base.js' `resolve` -> `alias`, which itself is a bit hard to maintain